### ISSUE TITLE
Add support for setting a runtime config preset in CLI option

### ISSUE THREAD

**Issue by teroyks** [valohai/valohai-cli#303]
Add an option (`--k8s-preset PRESET_UUID`) for `execution run` that will add the preset to the API call.

**Comment by axelauvinen** [comment]
Preset UUID is bit hidden currently, why not slug?

**Comment by teroyks** [comment]
> Preset UUID is bit hidden currently, why not slug?

It’s out of scope for this repository for now – API support for preset slugs is planned but not implemented yet in Roi, and that is needed before adding it here.

In other words: on it, but only after this MVP is in use. 🙂
